### PR TITLE
docs: fix broken links in Run Ingress on Amazon EKS blog

### LINKED ISSUE #325: docs: link is broken
**Actual Behaviour**

 There are some broken links in **ingress** docs.**_What about fixing these issues for now  or we could wait until  Markdown Link Checker is back??_**

Would you like to work on the issue?
**yes i would like to work on this issue**

Inked9_LI
Inked18_LI

**_What about fixing these issues for now  or we could wait until  Markdown Link Checker is back??_**

### PR DESCRIPTION
**Pull request by KishaniKandasamy**
Fixes: #325 

Changes:
update run-ingress-apisix-on-amazon-eks

### CODE REVIEW

**Comment by tokers** [comment]
@KishaniKandasamy Do the other documentations (like ACK, TKE, GKE) also have these problems?

**Comment by KishaniKandasamy** [comment]
> @KishaniKandasamy Do the other documentations (like ACK, TKE, GKE) also have these problems?

No. 
Just checked.

**Review by Yiyiyimu — APPROVED** [review]

File: website/blog/2021-01-21-run-ingress-apisix-on-amazon-eks.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ authorImageURL: "https://avatars0.githubusercontent.com/u/10428333?s=60&v=4"
 > - https://github.com/apache/apisix-helm-chart
 > - https://github.com/apache/apisix-ingress-controller
 
-This post is based on [Install Ingress APISIX on Amazon EKS](https://github.com/apache/apisix-ingress-controller/blob/master/docs/deployments/aws.md).
+This post is based on [Install Ingress APISIX on Amazon EKS](https://github.com/apache/apisix-ingress-controller/blob/master/docs/en/latest/deployments/aws.md).
```
Comment (Yiyiyimu) [inline website/blog/2021-01-21-run-ingress-apisix-on-amazon-eks.md]: might be better to use the website link
http://apisix.incubator.apache.org/docs/ingress-controller/deployments/aws/

**Review by tokers — APPROVED** [review]

**Comment by Yiyiyimu** [comment]
@KishaniKandasamy Sorry I provide the wrong URL, it should be `http://apisix.apache.org/docs/ingress-controller/deployments/aws/`. Do you mind fixing it again 😿

**Comment by KishaniKandasamy** [comment]
> @KishaniKandasamy Sorry I provide the wrong URL, it should be `http://apisix.apache.org/docs/ingress-controller/deployments/aws/`. Do you mind fixing it again 😿

No problem

File: website/blog/2021-01-21-run-ingress-apisix-on-amazon-eks.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ kubectl get service apisix-gateway \
 
  Another thing should be concerned that the `allow.ipList` field should be customized according to the [EKS CIDR Ranges](https://amazonaws-china.com/premiumsupport/knowledge-center/eks-multiple-cidr-ranges/) in your EKS cluster, so that the apisix-ingress-controller can be authorized by Apache APISIX (for the resources pushing).
 
-See value.yaml](https://github.com/apache/apisix-helm-chart/blob/master/chart/apisix/values.yaml) to learn all the configuration items if you have other requirements.
+See value.yaml](https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix/values.yaml) to learn all the configuration items if you have other requirements.
```
Comment (Yiyiyimu) [inline website/blog/2021-01-21-run-ingress-apisix-on-amazon-eks.md]: Missing a bracket here

File: website/blog/2021-01-21-run-ingress-apisix-on-amazon-eks.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ authorImageURL: "https://avatars0.githubusercontent.com/u/10428333?s=60&v=4"
 > - https://github.com/apache/apisix-helm-chart
 > - https://github.com/apache/apisix-ingress-controller
 
-This post is based on [Install Ingress APISIX on Amazon EKS](https://github.com/apache/apisix-ingress-controller/blob/master/docs/en/latest/deployments/aws.md).
+This post is based on [Install Ingress APISIX on Amazon EKS](http://apisix.incubator.apache.org/docs/ingress-controller/deployments/aws/).
```
Comment (tokers) [inline website/blog/2021-01-21-run-ingress-apisix-on-amazon-eks.md]:
```suggestion
This post is based on [Install Ingress APISIX on Amazon EKS](https://apisix.apache.org/docs/ingress-controller/deployments/aws/).
```

Comment (Yiyiyimu) [reply website/blog/2021-01-21-run-ingress-apisix-on-amazon-eks.md]: already fixed

**Pull request merged**